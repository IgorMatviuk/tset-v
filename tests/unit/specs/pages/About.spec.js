import { shallowMount } from '@vue/test-utils'

import About from '@/pages/-about.vue'

describe('About page', () => {
  test('has expected structure', () => {
    const wrapper = shallowMount(About, {
      mocks: {
        $t: (str) => str,
        $i18n: {
          path: (str) => str
        },
        $store: {
          state: {
            breadcrumb: []
          }
        }
      }
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
