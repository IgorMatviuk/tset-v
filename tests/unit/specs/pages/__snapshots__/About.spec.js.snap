// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page has expected structure 1`] = `
<main
  class="about-page"
>
  <thebreadcrumb-stub
    isfloat="true"
    items="[object Object],[object Object]"
  />
   
  <blockjumbotron-stub
    class="about-page__presentation"
    content="[object Object]"
  >
    <div>
      aboutPage.title
    </div>
     
    <div>
      aboutPage.subtitle
    </div>
  </blockjumbotron-stub>
   
  <blockaboutcontent-stub
    class="about-page__who-we"
    content="[object Object]"
    direction="right"
    isfirst="true"
    type="overlay"
  />
   
  <blockblockquote-stub
    class="about-page__blockquote"
    content="Мир моды соткан из историй.<br><b>Теперь мы творим свою собственную,</b> и приглашаем Вас <b>рассказать эту историю вместе с нами.</b>"
  />
   
  <blockaboutcontent-stub
    class="about-page__production"
    content="[object Object]"
    direction="left"
    type="overlay"
  />
   
  <blockourfeatures-stub
    class="about-page__features"
    features="[object Object],[object Object],[object Object],[object Object]"
  />
   
  <blockaboutcontent-stub
    class="about-page__history"
    content="[object Object]"
    direction="right"
    type="background"
  />
   
  <blockaboutcontent-stub
    class="about-page__team"
    content="[object Object]"
    direction="left"
    type="stretch"
  />
</main>
`;
